tests/warn/i5013.scala

```diff
@@ -0,0 +1,11 @@
+
+
+class Foo {
```

Comment:
For files where git cannot track the change, we might just remove the empty lines at the start. However, this could cause some additional check files to not be tracked.

Reply:
> this could cause some additional check files to not be tracked

What do you mean?